src/lib.rs

```diff
@@ -123,6 +123,9 @@ mod init;
 pub mod pbkdf2;
 
 pub mod rand;
+mod limb;
+pub use limb::GFp_rand_mod;
```

Comment:
OK, thanks for correcting me. Please just add a comment where the `pub use` is and ensure that we have the `#[doc(hidden)]` right so that it doesn't show up in the docs.